src/components/appNavigation/PushButton.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import React from 'react'
+import CustomButton, { type ButtonProps } from '../common/buttons/CustomButton'
+
+type PushButtonProps = {
+  ...ButtonProps,
+  routeName: Route,
+  params?: any,
+  screenProps: { push: (routeName: string, params: any) => void },
+  canContinue?: Function
+}
+
+/**
+ * PushButton
+ * This button gets the push action from screenProps. Is meant to be used inside a stackNavigator
+ * @param routeName
+ * @param screenProps
+ * @param params
+ * @param {ButtonProps} props
+ */
+export const PushButton = ({ routeName, screenProps, canContinue, params, ...props }: PushButtonProps) => {
```

Comment:
I would export this as default and move the component to /common/buttons @kbardi

Reply:
> @kbardi, when the app starts I have a kind of placeholder feed empty item.
> 
> It's allowing me to swipe it...
> ![swipe-feed](https://user-images.githubusercontent.com/3315606/61000273-7601cd80-a333-11e9-8780-ed3cf454255a.gif)
> 
> ... and to click it
> ![clickable-feed](https://user-images.githubusercontent.com/3315606/61000374-a77a9900-a333-11e9-9435-0f58a66be6e6.gif)
> 
> is it ok for this first iteration?

I cannot disable swipe, but I disabled click over empty feed and I added a white background instead of the red one when you swipe an empty feed